### PR TITLE
add optional resource limits to Machinaris Chart on `test` track

### PR DESCRIPTION
**Pull request by g-a-c**
I was having trouble with memory usage on my low-powered NAS, so this allows me to specify a CPU limit and memory limit (in Kubernetes syntax, e.g. `500m` for 500 millicores and `1Gi` for 1 Gibibyte of RAM), and I thought this might be useful for others so committed it to the `test` track initially

### CODE REVIEW

File: test/machinaris/1.0.6/questions.yaml [hunk 1]
```diff
@@ -157,3 +159,20 @@ questions:
                 label: "Value"
                 schema:
                   type: string
+  - variable: enableResourceLimits
+    label: "Enable Pod resource limits?"
```
Comment (sonicaj) [inline test/machinaris/1.0.6/questions.yaml:163]: Let's please remove the question mark

File: test/machinaris/1.0.6/questions.yaml [hunk 2]
```diff
@@ -157,3 +159,20 @@ questions:
...
+    group: "Resource Limits"
+    schema:
+      type: boolean
+  - variable: cpuLimit
+    label: "CPU limit"
+    group: "Resource Limits"
+    schema:
+      type: string
```
Comment (sonicaj) [inline test/machinaris/1.0.6/questions.yaml:171]: Do you think we should perhaps validate if specified value is valid using regex https://github.com/truenas/charts/blob/master/docs/schema/string.md ?

**Review by sonicaj — COMMENTED** [review]
Thank you for your contribution, just a minor nit

File: test/machinaris/1.0.6/questions.yaml (hunk 2 above)
Comment (g-a-c) [reply test/machinaris/1.0.6/questions.yaml:171]: Both values should now be regex-validated, the memory value comes straight from within the Kubernetes/`k3s` codebase, the CPU value is mine but has been tested with [Regex101](https://regex101.com/r/0PQNCH/1) to make sure it passes and fails on expected values

**Review by sonicaj — APPROVED** [review]
Thanks!

**Pull request merged**